docs/release-notes/release-notes-0.18.0.md

```diff
@@ -188,6 +188,9 @@
   [Deprecate bumpclosefee for bumpforceclosefee and add `max_fee_rate` option
    to `closechannel` cmd](https://github.com/lightningnetwork/lnd/pull/8350).
 
+* Generate man pages automatically using `lncli generatemanpage` command for
+  both `lncli` and `lnd` commands when running `make install` in the Makefile (https://github.com/lightningnetwork/lnd/pull/8525).
```

Comment:
nit: see how the linking is done in other entries here. Make some part of the text clickable instead of just giving the raw link

Reply:
Thanks! Addressed